docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,129 @@
+# ADR 008: Decentralized Computer Emergency Response Team (dCERT) Group
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
+information in an emergency scenario, we propose the creation of a
+specialization group named The Decentralized Computer Emergency Response Team
+(dCERT).  Initially this group's role is intended to serve as coordinators
+between the validators, bug-hunters, developers, and other members of the wider
+community.  During a crisis time, the dCERT group would facilitate and relay
+input from a variety of stakeholders to the developers who are actively
+devising a patch to the software, this way sensitive information does not need
+to be publicly disclosed while some input from the community can still be
+gained. 
+
+Additionally, a special privilege is proposed for the dCERT group:
+the capacity to "circuit-break" (aka. temporarily disable)  a particular
+message path. 
+
+In the future it is foreseeable that the community may wish to expand the roles
+of dCERT with further responsibilities such as the capacity to "pre-approve" a
+security update hard-fork on behalf of the community prior to a full community
+wide vote whereby the sensitive information would be revealed prior to a
+vulnerability being patched on the live network.  
+
+## Decision
+
+The dCERT group is proposed to include an implementation of a `SpecializationGroup`
+as defined in [ADR 007](./adr-007-specialization-groups.md). This will include the 
+implementation of: 
+ - continuous voting
+ - slashing due to breach of soft contract
+ - revoking a member due to breach of soft contract
+ - emergency disband of the entire dCERT group (ex. for colluding maliciously) 
+ - compensation stipend from the community pool or other means decided by
+   governance
+
+This system necessitates the following new parameters: 
+ - blockly stipend allowance per dCERT member (suggested ~$10K/yr of atoms)
+ - maximum number of dCERT members (suggested 7) 
+ - required staked slashable tokens for each dCERT member (suggested $10K of atoms)
+ - unbonding time for dCERT staked token (suggested 3 week)
+ - quorum for suspending a particular member (suggested 5/7) 
+ - proposal wager for disbanding the dCERT group (suggested ~$100000 USD worth of atoms)
+
+These parameters are expected to be implemented through the param keeper such 
+that governance may change them at any given point. 
+
+### Continuous Voting Electionator
+
+An `Electionator` object is to be implemented with continuous voting is to be
+implemented with the following specifications:
+ - All delegation addresses may submit votes at any point which updates their 
+   preferred representation on the dCERT group. 
+ - Preferred representation may be arbitrarily split between address (ex. 50%
+   to John, 25% to Sally, 25% to Carol) 
+ - Addresses which control the greatest amount of preferred-representation are
```

Comment:
How do you handle splits?

Reply:
What happens if the community is split between two candidates at the lowest position and it turns contentious.

Both sides continue adding votes to one side of the election or the another, making the last position in the dCERT group see-saw between both candidates frequently (maybe even every couple of blocks). If there's a continuous election, i could see this potentially happening

Reply:
@AdityaSripal good point we could add a stabilizing time period, such that you can only be replaced if you've been on dCERT for one month. with would, in the worst case produce a 1 month see-saw

Reply:
We should amend to include this

Reply:
yeah I'm going to add into the ADR with an enable parameter, such that this stabilizer could be enabled be governance post-mortem

Reply:
Added this discussion!